### ISSUE TITLE
Doc at https://docs.rs/tokio-retry are out of date

### ISSUE THREAD

**Issue by simon-an** [srijs/rust-tokio-retry#19]
The example at https://docs.rs/tokio-retry/0.2.0/tokio_retry/index.html  show a code example which does not compile.
The "then" function does not exist on type retry.

```rust
let future = Retry::spawn(retry_strategy, action).then(|result| {
    println!("result {:?}", result);
    Ok(())
});
```

**Comment by apotukareon** [comment]
Current examples are not up-to-date, compare with tests: https://github.com/srijs/rust-tokio-retry/blob/master/tests/future.rs

**Comment by srijs** [comment]
Hi, thanks for the report!

I've just published version 0.3.0 based on tokio 1.0, which has updated examples that work: https://crates.io/crates/tokio-retry

The docs on docs.rs should be updated in a couple of minutes as well!
